frontend/src/components/program/picasso/Picasso.vue

```diff
@@ -104,15 +104,9 @@ Listing all given activity schedule entries in a calendar view.
       </template>
     </v-calendar>
 
-    <v-snackbar v-model="isSaving" light :color="patchError ? 'orange darken-2' : ''">
-      <template v-if="patchError">
-        <v-icon>mdi-alert</v-icon>
-        {{ patchError }}
-      </template>
```

Comment:
I thought it's better to display the errors directly in the component if possible, so why replace this here with a toast?

Reply:
Was an order from @usu 
Like this, we can display the last 2 errors which happened.
The previous toast only supported one toast.